src/main/java/io/wispforest/owo/ui/core/OwoUIDrawContext.java

```diff
@@ -168,16 +178,51 @@ public void drawText(Text text, float x, float y, float scale, int color, TextAn
             }
         }
 
-
-        this.drawText(textRenderer, text, (int) (x * (1 / scale)), (int) (y * (1 / scale)), color, false);
+        this.drawText(textRenderer, text, (int) (x * (1 / scale)), (int) (y * (1 / scale)), color, textShadow);
         this.getMatrices().pop();
     }
 
     public enum TextAnchor {
         TOP_RIGHT, BOTTOM_RIGHT, TOP_LEFT, BOTTOM_LEFT
     }
 
+    public void drawScrollableText(Text text, int startX, int startY, int endX, int endY, float scale, int color) {
+        drawScrollableText(text, (startX + endX) / 2, startX, startY, endX, endY, scale, color, false);
+    }
+
+    public void drawScrollableText(Text text, int startX, int startY, int endX, int endY, float scale, int color, boolean textShadow) {
+        drawScrollableText(text, (startX + endX) / 2, startX, startY, endX, endY, scale, color, textShadow);
+    }
+
+    public void drawScrollableText(Text text, int centerX, int startX, int startY, int endX, int endY, float scale, int color) {
+        drawScrollableText(text, centerX, startX, startY, endX, endY, scale, color, false);
+    }
+
+    public void drawScrollableText(Text text, int centerX, int startX, int startY, int endX, int endY, float scale, int color, boolean textShadow) {
```

Comment:
Is there a specific reason this code was copy-pasted from `ClickableWidget` instead of just calling the vanilla method?

Reply:
The `ClickableWidget` method is protected by default, enforces shadowed text and doesn't support scaling